tests/test_application_form.py

```diff
@@ -106,8 +106,8 @@ def test_form_pages_accessible(self):
                     },
                     route_rel,
                 )
-                assert len(results["violations"]) <= 2
-                assert len(results["violations"]) == 0 or all(
+                assert len(results["violations"]) <= 4
```

Comment:
Might be worth having a configuration file where stuff like this is specified - in the future we then have a single source of truth for where we are allowing problems through our pipeline.